crates/turborepo-lib/src/run/mod.rs

```diff
@@ -322,6 +289,27 @@ impl<'a> Run<'a> {
             &self.base.repo_root,
         )?;
 
+        if opts.run_opts.parallel {
+            pkg_dep_graph = pkg_dep_graph.no_workspace_dependencies();
+            engine = self.build_engine(&pkg_dep_graph, &opts, &root_turbo_json, &filtered_pkgs)?;
+        }
+
+        if let Some(graph_opts) = opts.run_opts.graph {
+            match graph_opts {
+                GraphOpts::File(graph_file) => {
```

Comment:
just making sure i understand this arm... it's not actually writing the graph to a file yet right? I don't see the `engine` or `pkg_dep_graph` used in here

Reply:
Correct, one of the features left is support for non-`DOT` `--graph` support. The `todo` macro call in this branch will panic when hit.